pymc3/distributions/continuous.py

```diff
@@ -1081,14 +1154,18 @@ class ExGaussian(Continuous):
         Tutorials in Quantitative Methods for Psychology,
         Vol. 4, No. 1, pp 35-45.
     """
-    def __init__(self, mu, sigma, nu, *args, **kwargs):
-        super(ExGaussian, self).__init__(*args, **kwargs)
-        self.mu = mu
-        self.sigma = sigma
-        self.nu = nu
+
+    def __init__(self, mu, sigma, nu, ndim=None, size=None, dtype=None, *args, **kwargs):
```

Comment:
I want to say that each `Distribution` instance's constructor should supply it, in the same way this PR requires them to specify their support, etc.